gym/spaces/dict.py

```diff
@@ -60,6 +60,9 @@ def contains(self, x):
                 return False
         return True
 
+    def __getitem__(self, key):
+        return self.spaces[key]
```

Comment:
What if `key` is None or does not exist in `self.spaces`? This will break.

Reply:
I think if key does not exist in self.spaces it should break - similarly to trying to get the non-existent element from a dictionary. As for `key == None` - why would that be a problem?